src/plugins/home/public/index.ts

```diff
@@ -21,13 +21,9 @@ import { PluginInitializerContext } from 'kibana/public';
 
 export {
   FeatureCatalogueSetup,
```

Comment:
If you're exposing these sub-service contract, maybe expose them as `FeatureCatalogueSetup as FeatureCatalogue`?

Reply:
Good point, I will fix that